modular_skyrat/modules/blastwave_outfits/code/clothing/goner_uniform.dm

```diff
@@ -0,0 +1,41 @@
+/obj/item/clothing/under/goner
```

Comment:
If it's called a blastwave uniform, then the typepath should be `/obj/item/clothing/under/blastwave`, otherwise it becomes annoying to find when spawning stuff in-game. (this goes for all of these).

That, or you just rename them all to goner instead, but I think blastwave sounds cooler, that being said I still haven't looked at the sprites.